client/src/components/ChatMessage.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface Props {
+  text: string;
+}
+
+const messageStyle = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '200px',
+  alignItems: 'flex-end',
+  padding: '10px',
+  borderRadius: '10px',
+  backgroundColor: '#09f',
+  margin: '10px'
+} as any;
```

Comment:
Let's please not do that. Using the any type basically defeats the purpose of TypeScript. Could you provide the error and then we can work through it?